toolkit/scripts/toolchain/cross/gcc_cross_test.sh

```diff
@@ -0,0 +1,53 @@
+# This script assumes that gcc_cross.sh has been run with the gcc extracted sources at $HOME/cross/gcc-9.1.0 and the gcc cross compilation binaries at /opt/cross/bin
```

Comment:
Copyright?
What is the purpose fo this script?
Note that the comment says gcc-9.1.0 but Mariner is on 11.2.  Is the version relevant?